server/applications.json

```diff
@@ -876,7 +876,7 @@
         "harmony": {
             "enabled": true,
             "host_name": "harmony",
-            "environment": "{\n    \"AYON_HARMONY_WORKFILES_ON_LAUNCH\": \"1\"\n}",
+            "environment": "",
```

Comment:
```suggestion
            "environment": "{}",
```

Reply:
Server actually handles that, but agree that this is cleaner.